.github/workflows/deploy.yml

```diff
@@ -46,24 +46,22 @@ jobs:
       run: python manage.py migrate
     - name: Run tests
       run: python manage.py test
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
     - name: Docker meta
       id: meta
       uses: docker/metadata-action@v3
       with:
-        images: quay.io/wisvch/dienst2
+        images: ghcr.io/wisvch/dienst2
         tags: type=sha, prefix={{date 'YYYYMMDD'}}-
-    - name: Set up Docker Buildx
```

Comment:
Why is this being removed? It is still [recommended by GitHub](https://github.com/marketplace/actions/build-and-push-docker-images#usage).

Reply:
readded it